### ISSUE TITLE
Render icon and menu when app is started in an offline state

### ISSUE THREAD

**Issue by adamwatters** [stefanjudis/netlify-menubar#13]
The initial render of the app is held up until an online connection is detected. I think the expected UX is for the app to visibly start up even when offline.

how it looks now
![image](https://user-images.githubusercontent.com/7673611/52892950-812a4600-3165-11e9-8054-386a3df0c347.png)

how I think it should look
![image](https://user-images.githubusercontent.com/7673611/52892759-ac139a80-3163-11e9-9012-9559eaa4cd0b.png)

**Comment by stefanjudis** [comment]
I agree here but dont think its a big issue though. I went the easy route to not bloat the code. 🙈

Happy to take fixes. 😊

**Comment by adamwatters** [comment]
of course! definitely not a big issue. just flagging it as another place to tinker after I finish https://github.com/stefanjudis/netlify-menubar/issues/11

**Comment by stefanjudis** [comment]
Looking forward for your contribution! hopefully the code is not too messy. TypeScript gave me a hard time. 🙈

**Comment by adamwatters** [comment]
lol don't sweat it - it's my first time with TypeScript too!
